src/pages/Host/Host.js

```diff
@@ -1,5 +1,196 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import HostHouse from './HostStair/HostHouse';
+import HostMain from './HostStair/HostMain';
+import HostMap from './HostStair/HostMap';
+import HostBed from './HostStair/HostBed';
+import HostPicture from './HostStair/HostPicture';
+import HostInfo from './HostStair/HostInfo';
+import HostPrice from './HostStair/HostPrice';
+import HostVideo from './HostMessage/HostVideo';
+import PhotoVideo from './HostMessage//PhotoVideo';
+import PriceVideo from './HostMessage//PriceVideo';
+import variables from '../../styles/variables';
+import Mapmessage from './HostMessage/MapMessage';
+import HouseMessage from './HostMessage/HouseMessage';
+import InfoMessage from './HostMessage/InfoMessage';
+import Locationmessage from './HostMessage/Locationmessage';
+
 const Host = () => {
-  return <>호스트</>;
+  const [pageCount, setPageCount] = useState(0);
+  const navigate = useNavigate();
+  const check = () => {
+    alert('입력이 완료되었습니다. 😄');
+    navigate('/hostresult');
+  };
+  const nextPage = () => {
+    pageCount === 6 ? check() : setPageCount(prev => prev + 1);
+  };
+  const backPage = () => {
+    pageCount === 0 ? setPageCount(6) : setPageCount(prev => prev - 1);
+  };
+
+  const nextGoPage = [
+    <HostMain />,
+    <HostHouse />,
+    <HostMap />,
+    <HostBed />,
+    <HostPicture />,
+    <HostInfo />,
+    <HostPrice />,
+  ];
+  const message = [
+    <HostVideo />,
+    <Locationmessage />,
+    <Mapmessage />,
+    <HouseMessage />,
+    <PhotoVideo />,
+    <InfoMessage />,
+    <PriceVideo />,
+  ];
+
+  const goMain = () => {
+    navigate('/');
+  };
+
+  return (
+    <S.HostBox>
+      <S.HostBoxCenter>
+        <S.HostVideoBox>
+          <S.MessageBox>{message[pageCount]}</S.MessageBox>
+        </S.HostVideoBox>
+        <S.HostQuestion>
+          <S.HostQuestionQuitBox>
+            <S.HostQuestionQuit onClick={goMain}>나가기</S.HostQuestionQuit>
+          </S.HostQuestionQuitBox>
+          {nextGoPage[pageCount]}
+          <S.HostBar count={pageCount} />
+          <S.HostQuestionStartBox>
+            {pageCount === 0 ? (
+              <S.HiddenBox></S.HiddenBox>
+            ) : (
+              <S.HostBack onClick={backPage}>뒤로 </S.HostBack>
+            )}
+
+            <S.HostQuestionStart onClick={nextPage}>
+              {pageCount === 0 ? '시작하기' : '다음'}
+            </S.HostQuestionStart>
+          </S.HostQuestionStartBox>
+        </S.HostQuestion>
+      </S.HostBoxCenter>
+    </S.HostBox>
+  );
 };
 export default Host;
+
+const S = {
+  HostBox: styled.div`
+    ${variables.flex('row', 'center', 'center')};
```

Comment:
flex의 기본 값이 'row', 'center', 'center' 이기 때문에 이렇게 쓰면 더 좋을 것 같습니다. 
```suggestion
    ${variables.flex()};
```